gossipd/gossip.c

```diff
@@ -1769,6 +1769,28 @@ static struct io_plan *handle_txout_reply(struct io_conn *conn,
 	return daemon_conn_read_next(conn, &daemon->master);
 }
 
+static struct io_plan *routing_failure_req(struct io_conn *conn,
```

Comment:
Nit: `_req` suggests that there is a reply to this, just `routing_failure` should be fine.